AKARI/requirements.txt

```
@@ -0,0 +1,4 @@
+opencv-python
+pyzbar
+requests
+akari-client
```

```suggestion
akari-client[depthai]

```
depthai, blobconverterなどのライブラリが不足なので、上記を追加して下さい。